spec/features/item_view_metadata_spec.rb

```diff
@@ -1,17 +1,20 @@
 require 'spec_helper'
 
 feature 'Item view metadata' do
-  before do
-    @current_user = double(
+  let(:current_user) do
```

Comment:
Ah.. yes. I think the issue might be here with using in "Feature" spec while giving using the "any_instance_of" mock.